rav1e-by-gop/src/encode/mod.rs

```diff
@@ -98,7 +102,8 @@ fn do_encode<T: Pixel + DeserializeOwned>(
             opts.max_bitrate,
             video_info,
             source.frame_count(),
-        );
+        )
+        .with_thread_pool(pool.clone());
```

Comment:
Could you please apply the clippy lint here?
```suggestion
        .with_thread_pool(pool);
```